### DISCUSSION TITLE
How to append a exist parquet file

### DISCUSSION THREAD

**Discussion by attila-lin** [jorgecarleitao/arrow2#957] [Q&A] [answered]
The `WriteOptions` seems do not have this config.

After write few times, the file gets big while the contents is steal the same, I think it just write the append content to the end of the file will be ignored.

**Comment by jorgecarleitao** [comment, marked as answer]
see https://github.com/jorgecarleitao/parquet2/issues/47 - it is not supported by parquet2 (although not an impossible task). The main changes need to happen there - here we just create parquet pages
